Automated dev-it test failures with latest binary scanner jar

The following integration tests are failing from the dev-it suite around the time that the new binary scanner jar was uploaded to Sonatype:

<details>
 <summary>net.wasdev.wlp.test.dev.it.DevCopyTestDependenciesTest</summary>
```
Running net.wasdev.wlp.test.dev.it.DevCopyTestDependenciesTest
[INFO] Tests run: 2, Failures: 1, Errors: 1, Skipped: 0, Time elapsed: 0 sec <<< FAILURE! - in net.wasdev.wlp.test.dev.it.DevCopyTestDependenciesTest
[INFO] net.wasdev.wlp.test.dev.it.DevCopyTestDependenciesTest  Time elapsed: 0 sec  <<< FAILURE!
[INFO] java.lang.AssertionError
[INFO] 	at org.junit.Assert.fail(Assert.java:87)
[INFO] 	at org.junit.Assert.assertTrue(Assert.java:42)
[INFO] 	at org.junit.Assert.assertTrue(Assert.java:53)
[INFO] 	at net.wasdev.wlp.test.dev.it.BaseDevTest.setUpBeforeClass(BaseDevTest.java:111)
[INFO] 	at net.wasdev.wlp.test.dev.it.DevCopyTestDependenciesTest.setUpBeforeClass(DevCopyTestDependenciesTest.java:35)
[INFO] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[INFO] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[INFO] 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[INFO] 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
[INFO] 	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
[INFO] 	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
[INFO] 	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
[INFO] 	at org.junit.internal.runners.statements.RunBefores.invokeMethod(RunBefores.java:33)
[INFO] 	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:24)
[INFO] 	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
[INFO] 	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
[INFO] 	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
[INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:367)
[INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:274)
[INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:238)
[INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:161)
[INFO] 	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
[INFO] 	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
[INFO] 	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
[INFO] 
[INFO] net.wasdev.wlp.test.dev.it.DevCopyTestDependenciesTest  Time elapsed: 0 sec  <<< ERROR!
[INFO] java.io.IOException: Stream closed
[INFO] 	at java.base/java.io.BufferedWriter.ensureOpen(BufferedWriter.java:107)
[INFO] 	at java.base/java.io.BufferedWriter.write(BufferedWriter.java:224)
[INFO] 	at java.base/java.io.Writer.write(Writer.java:249)
[INFO] 	at net.wasdev.wlp.test.dev.it.BaseDevTest.stopProcess(BaseDevTest.java:244)
[INFO] 	at net.wasdev.wlp.test.dev.it.BaseDevTest.cleanUpAfterClass(BaseDevTest.java:221)
[INFO] 	at net.wasdev.wlp.test.dev.it.BaseDevTest.cleanUpAfterClass(BaseDevTest.java:217)
[INFO] 	at net.wasdev.wlp.test.dev.it.BaseDevTest.cleanUpAfterClass(BaseDevTest.java:213)
[INFO] 	at net.wasdev.wlp.test.dev.it.DevCopyTestDependenciesTest.cleanUpAfterClass(DevCopyTestDependenciesTest.java:48)
[INFO] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[INFO] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[INFO] 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[INFO] 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
[INFO] 	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
[INFO] 	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
[INFO] 	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
[INFO] 	at org.junit.internal.runners.statements.RunAfters.invokeMethod(RunAfters.java:46)
[INFO] 	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:33)
[INFO] 	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
[INFO] 	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
[INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:367)
[INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:274)
[INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:238)
[INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:161)
[INFO] 	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
[INFO] 	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
[INFO] 	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
[INFO] 
```
</details>

<details>
 <summary>net.wasdev.wlp.test.dev.it.DevCopyTestDependenciesVersionTest</summary>
```
[INFO] Running net.wasdev.wlp.test.dev.it.DevCopyTestDependenciesVersionTest
[INFO] Tests run: 2, Failures: 1, Errors: 1, Skipped: 0, Time elapsed: 0.005 sec <<< FAILURE! - in net.wasdev.wlp.test.dev.it.DevCopyTestDependenciesVersionTest
[INFO] net.wasdev.wlp.test.dev.it.DevCopyTestDependenciesVersionTest  Time elapsed: 0.005 sec  <<< FAILURE!
[INFO] java.lang.AssertionError
[INFO] 	at org.junit.Assert.fail(Assert.java:87)
[INFO] 	at org.junit.Assert.assertTrue(Assert.java:42)
[INFO] 	at org.junit.Assert.assertTrue(Assert.java:53)
[INFO] 	at net.wasdev.wlp.test.dev.it.BaseDevTest.setUpBeforeClass(BaseDevTest.java:111)
[INFO] 	at net.wasdev.wlp.test.dev.it.DevCopyTestDependenciesVersionTest.setUpBeforeClass(DevCopyTestDependenciesVersionTest.java:35)
[INFO] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[INFO] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[INFO] 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[INFO] 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
[INFO] 	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
[INFO] 	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
[INFO] 	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
[INFO] 	at org.junit.internal.runners.statements.RunBefores.invokeMethod(RunBefores.java:33)
[INFO] 	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:24)
[INFO] 	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
[INFO] 	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
[INFO] 	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
[INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:367)
[INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:274)
[INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:238)
[INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:161)
[INFO] 	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
[INFO] 	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
[INFO] 	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
[INFO] 
[INFO] net.wasdev.wlp.test.dev.it.DevCopyTestDependenciesVersionTest  Time elapsed: 0.005 sec  <<< ERROR!
[INFO] java.io.IOException: Stream closed
[INFO] 	at java.base/java.io.BufferedWriter.ensureOpen(BufferedWriter.java:107)
[INFO] 	at java.base/java.io.BufferedWriter.write(BufferedWriter.java:224)
[INFO] 	at java.base/java.io.Writer.write(Writer.java:249)
[INFO] 	at net.wasdev.wlp.test.dev.it.BaseDevTest.stopProcess(BaseDevTest.java:244)
[INFO] 	at net.wasdev.wlp.test.dev.it.BaseDevTest.cleanUpAfterClass(BaseDevTest.java:221)
[INFO] 	at net.wasdev.wlp.test.dev.it.BaseDevTest.cleanUpAfterClass(BaseDevTest.java:217)
[INFO] 	at net.wasdev.wlp.test.dev.it.BaseDevTest.cleanUpAfterClass(BaseDevTest.java:213)
[INFO] 	at net.wasdev.wlp.test.dev.it.DevCopyTestDependenciesVersionTest.cleanUpAfterClass(DevCopyTestDependenciesVersionTest.java:48)
[INFO] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[INFO] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[INFO] 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[INFO] 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
[INFO] 	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
[INFO] 	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
[INFO] 	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
[INFO] 	at org.junit.internal.runners.statements.RunAfters.invokeMethod(RunAfters.java:46)
[INFO] 	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:33)
[INFO] 	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
[INFO] 	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
[INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:367)
[INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:274)
[INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:238)
[INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:161)
[INFO] 	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
[INFO] 	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
[INFO] 	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
[INFO] 
```
</details>

<details>
 <summary>net.wasdev.wlp.test.dev.it.DevRecompileDepsTest</summary>
```
[INFO] Running net.wasdev.wlp.test.dev.it.DevRecompileDepsTest
[INFO] Tests run: 2, Failures: 1, Errors: 1, Skipped: 0, Time elapsed: 0.007 sec <<< FAILURE! - in net.wasdev.wlp.test.dev.it.DevRecompileDepsTest
[INFO] net.wasdev.wlp.test.dev.it.DevRecompileDepsTest  Time elapsed: 0.007 sec  <<< FAILURE!
[INFO] java.lang.AssertionError
[INFO] 	at org.junit.Assert.fail(Assert.java:87)
[INFO] 	at org.junit.Assert.assertTrue(Assert.java:42)
[INFO] 	at org.junit.Assert.assertTrue(Assert.java:53)
[INFO] 	at net.wasdev.wlp.test.dev.it.BaseDevTest.setUpBeforeClass(BaseDevTest.java:111)
[INFO] 	at net.wasdev.wlp.test.dev.it.BaseDevTest.setUpBeforeClass(BaseDevTest.java:80)
[INFO] 	at net.wasdev.wlp.test.dev.it.BaseDevTest.setUpBeforeClass(BaseDevTest.java:76)
[INFO] 	at net.wasdev.wlp.test.dev.it.BaseDevTest.setUpBeforeClass(BaseDevTest.java:68)
[INFO] 	at net.wasdev.wlp.test.dev.it.DevRecompileDepsTest.setUpBeforeClass(DevRecompileDepsTest.java:27)
[INFO] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[INFO] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[INFO] 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[INFO] 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
[INFO] 	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
[INFO] 	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
[INFO] 	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
[INFO] 	at org.junit.internal.runners.statements.RunBefores.invokeMethod(RunBefores.java:33)
[INFO] 	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:24)
[INFO] 	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
[INFO] 	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
[INFO] 	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
[INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:367)
[INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:274)
[INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:238)
[INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:161)
[INFO] 	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
[INFO] 	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
[INFO] 	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
[INFO] 
[INFO] net.wasdev.wlp.test.dev.it.DevRecompileDepsTest  Time elapsed: 0.007 sec  <<< ERROR!
[INFO] java.io.IOException: Stream closed
[INFO] 	at java.base/java.io.BufferedWriter.ensureOpen(BufferedWriter.java:107)
[INFO] 	at java.base/java.io.BufferedWriter.write(BufferedWriter.java:224)
[INFO] 	at java.base/java.io.Writer.write(Writer.java:249)
[INFO] 	at net.wasdev.wlp.test.dev.it.BaseDevTest.stopProcess(BaseDevTest.java:244)
[INFO] 	at net.wasdev.wlp.test.dev.it.BaseDevTest.cleanUpAfterClass(BaseDevTest.java:221)
[INFO] 	at net.wasdev.wlp.test.dev.it.BaseDevTest.cleanUpAfterClass(BaseDevTest.java:217)
[INFO] 	at net.wasdev.wlp.test.dev.it.BaseDevTest.cleanUpAfterClass(BaseDevTest.java:213)
[INFO] 	at net.wasdev.wlp.test.dev.it.DevRecompileDepsTest.cleanUpAfterClass(DevRecompileDepsTest.java:32)
[INFO] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[INFO] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[INFO] 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[INFO] 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
[INFO] 	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
[INFO] 	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
[INFO] 	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
[INFO] 	at org.junit.internal.runners.statements.RunAfters.invokeMethod(RunAfters.java:46)
[INFO] 	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:33)
[INFO] 	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
[INFO] 	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
[INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:367)
[INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:274)
[INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:238)
[INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:161)
[INFO] 	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
[INFO] 	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
[INFO] 	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
[INFO] 
```
</details>

<details>
 <summary>net.wasdev.wlp.test.dev.it.DevCopyProvidedDependenciesVersionTest</summary>
```
[INFO] Running net.wasdev.wlp.test.dev.it.DevCopyProvidedDependenciesVersionTest
[INFO] Tests run: 2, Failures: 1, Errors: 1, Skipped: 0, Time elapsed: 0.004 sec <<< FAILURE! - in net.wasdev.wlp.test.dev.it.DevCopyProvidedDependenciesVersionTest
[INFO] net.wasdev.wlp.test.dev.it.DevCopyProvidedDependenciesVersionTest  Time elapsed: 0.003 sec  <<< FAILURE!
[INFO] java.lang.AssertionError
[INFO] 	at org.junit.Assert.fail(Assert.java:87)
[INFO] 	at org.junit.Assert.assertTrue(Assert.java:42)
[INFO] 	at org.junit.Assert.assertTrue(Assert.java:53)
[INFO] 	at net.wasdev.wlp.test.dev.it.BaseDevTest.setUpBeforeClass(BaseDevTest.java:111)
[INFO] 	at net.wasdev.wlp.test.dev.it.DevCopyProvidedDependenciesVersionTest.setUpBeforeClass(DevCopyProvidedDependenciesVersionTest.java:35)
[INFO] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[INFO] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[INFO] 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[INFO] 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
[INFO] 	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
[INFO] 	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
[INFO] 	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
[INFO] 	at org.junit.internal.runners.statements.RunBefores.invokeMethod(RunBefores.java:33)
[INFO] 	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:24)
[INFO] 	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
[INFO] 	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
[INFO] 	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
[INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:367)
[INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:274)
[INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:238)
[INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:161)
[INFO] 	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
[INFO] 	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
[INFO] 	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
[INFO] 
[INFO] net.wasdev.wlp.test.dev.it.DevCopyProvidedDependenciesVersionTest  Time elapsed: 0.004 sec  <<< ERROR!
[INFO] java.io.IOException: Stream closed
[INFO] 	at java.base/java.io.BufferedWriter.ensureOpen(BufferedWriter.java:107)
[INFO] 	at java.base/java.io.BufferedWriter.write(BufferedWriter.java:224)
[INFO] 	at java.base/java.io.Writer.write(Writer.java:249)
[INFO] 	at net.wasdev.wlp.test.dev.it.BaseDevTest.stopProcess(BaseDevTest.java:244)
[INFO] 	at net.wasdev.wlp.test.dev.it.BaseDevTest.cleanUpAfterClass(BaseDevTest.java:221)
[INFO] 	at net.wasdev.wlp.test.dev.it.BaseDevTest.cleanUpAfterClass(BaseDevTest.java:217)
[INFO] 	at net.wasdev.wlp.test.dev.it.BaseDevTest.cleanUpAfterClass(BaseDevTest.java:213)
[INFO] 	at net.wasdev.wlp.test.dev.it.DevCopyProvidedDependenciesVersionTest.cleanUpAfterClass(DevCopyProvidedDependenciesVersionTest.java:48)
[INFO] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[INFO] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[INFO] 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[INFO] 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
[INFO] 	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
[INFO] 	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
[INFO] 	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
[INFO] 	at org.junit.internal.runners.statements.RunAfters.invokeMethod(RunAfters.java:46)
[INFO] 	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:33)
[INFO] 	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
[INFO] 	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
[INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:367)
[INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:274)
[INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:238)
[INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:161)
[INFO] 	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
[INFO] 	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
[INFO] 	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
[INFO] 
```
</details>

<details>
 <summary>net.wasdev.wlp.test.dev.it.DevHotTestingTest</summary>
```
[INFO] Tests run: 2, Failures: 1, Errors: 1, Skipped: 0, Time elapsed: 0.005 sec <<< FAILURE! - in net.wasdev.wlp.test.dev.it.DevHotTestingTest
[INFO] net.wasdev.wlp.test.dev.it.DevHotTestingTest  Time elapsed: 0.004 sec  <<< FAILURE!
[INFO] java.lang.AssertionError
[INFO] 	at org.junit.Assert.fail(Assert.java:87)
[INFO] 	at org.junit.Assert.assertTrue(Assert.java:42)
[INFO] 	at org.junit.Assert.assertTrue(Assert.java:53)
[INFO] 	at net.wasdev.wlp.test.dev.it.BaseDevTest.setUpBeforeClass(BaseDevTest.java:111)
[INFO] 	at net.wasdev.wlp.test.dev.it.BaseDevTest.setUpBeforeClass(BaseDevTest.java:80)
[INFO] 	at net.wasdev.wlp.test.dev.it.BaseDevTest.setUpBeforeClass(BaseDevTest.java:76)
[INFO] 	at net.wasdev.wlp.test.dev.it.BaseDevTest.setUpBeforeClass(BaseDevTest.java:68)
[INFO] 	at net.wasdev.wlp.test.dev.it.DevHotTestingTest.setUpBeforeClass(DevHotTestingTest.java:27)
[INFO] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[INFO] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[INFO] 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[INFO] 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
[INFO] 	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
[INFO] 	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
[INFO] 	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
[INFO] 	at org.junit.internal.runners.statements.RunBefores.invokeMethod(RunBefores.java:33)
[INFO] 	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:24)
[INFO] 	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
[INFO] 	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
[INFO] 	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
[INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:367)
[INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:274)
[INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:238)
[INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:161)
[INFO] 	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
[INFO] 	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
[INFO] 	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
[INFO] 
[INFO] net.wasdev.wlp.test.dev.it.DevHotTestingTest  Time elapsed: 0.005 sec  <<< ERROR!
[INFO] java.io.IOException: Stream closed
[INFO] 	at java.base/java.lang.ProcessBuilder$NullOutputStream.write(ProcessBuilder.java:442)
[INFO] 	at java.base/java.io.OutputStream.write(OutputStream.java:157)
[INFO] 	at java.base/java.io.BufferedOutputStream.flushBuffer(BufferedOutputStream.java:81)
[INFO] 	at java.base/java.io.BufferedOutputStream.flush(BufferedOutputStream.java:142)
[INFO] 	at java.base/sun.nio.cs.StreamEncoder.implFlush(StreamEncoder.java:318)
[INFO] 	at java.base/sun.nio.cs.StreamEncoder.flush(StreamEncoder.java:153)
[INFO] 	at java.base/java.io.OutputStreamWriter.flush(OutputStreamWriter.java:251)
[INFO] 	at java.base/java.io.BufferedWriter.flush(BufferedWriter.java:257)
[INFO] 	at net.wasdev.wlp.test.dev.it.BaseDevTest.stopProcess(BaseDevTest.java:249)
[INFO] 	at net.wasdev.wlp.test.dev.it.BaseDevTest.cleanUpAfterClass(BaseDevTest.java:221)
[INFO] 	at net.wasdev.wlp.test.dev.it.BaseDevTest.cleanUpAfterClass(BaseDevTest.java:217)
[INFO] 	at net.wasdev.wlp.test.dev.it.BaseDevTest.cleanUpAfterClass(BaseDevTest.java:213)
[INFO] 	at net.wasdev.wlp.test.dev.it.DevHotTestingTest.cleanUpAfterClass(DevHotTestingTest.java:32)
[INFO] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[INFO] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[INFO] 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[INFO] 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
[INFO] 	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
[INFO] 	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
[INFO] 	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
[INFO] 	at org.junit.internal.runners.statements.RunAfters.invokeMethod(RunAfters.java:46)
[INFO] 	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:33)
[INFO] 	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
[INFO] 	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
[INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:367)
[INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:274)
[INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:238)
[INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:161)
[INFO] 	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
[INFO] 	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
[INFO] 	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
[INFO] 
```
</details>

<details>
 <summary>net.wasdev.wlp.test.dev.it.PollingDevTest</summary>
```
[INFO] Running net.wasdev.wlp.test.dev.it.PollingDevTest
[INFO] Tests run: 2, Failures: 1, Errors: 1, Skipped: 0, Time elapsed: 0.005 sec <<< FAILURE! - in net.wasdev.wlp.test.dev.it.PollingDevTest
[INFO] net.wasdev.wlp.test.dev.it.PollingDevTest  Time elapsed: 0.005 sec  <<< FAILURE!
[INFO] java.lang.AssertionError
[INFO] 	at org.junit.Assert.fail(Assert.java:87)
[INFO] 	at org.junit.Assert.assertTrue(Assert.java:42)
[INFO] 	at org.junit.Assert.assertTrue(Assert.java:53)
[INFO] 	at net.wasdev.wlp.test.dev.it.BaseDevTest.setUpBeforeClass(BaseDevTest.java:111)
[INFO] 	at net.wasdev.wlp.test.dev.it.BaseDevTest.setUpBeforeClass(BaseDevTest.java:80)
[INFO] 	at net.wasdev.wlp.test.dev.it.BaseDevTest.setUpBeforeClass(BaseDevTest.java:76)
[INFO] 	at net.wasdev.wlp.test.dev.it.BaseDevTest.setUpBeforeClass(BaseDevTest.java:68)
[INFO] 	at net.wasdev.wlp.test.dev.it.PollingDevTest.setUpBeforeClass(PollingDevTest.java:23)
[INFO] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[INFO] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[INFO] 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[INFO] 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
[INFO] 	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
[INFO] 	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
[INFO] 	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
[INFO] 	at org.junit.internal.runners.statements.RunBefores.invokeMethod(RunBefores.java:33)
[INFO] 	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:24)
[INFO] 	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
[INFO] 	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
[INFO] 	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
[INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:367)
[INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:274)
[INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:238)
[INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:161)
[INFO] 	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
[INFO] 	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
[INFO] 	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
[INFO] 
[INFO] net.wasdev.wlp.test.dev.it.PollingDevTest  Time elapsed: 0.005 sec  <<< ERROR!
[INFO] java.io.IOException: Stream closed
[INFO] 	at java.base/java.io.BufferedWriter.ensureOpen(BufferedWriter.java:107)
[INFO] 	at java.base/java.io.BufferedWriter.write(BufferedWriter.java:224)
[INFO] 	at java.base/java.io.Writer.write(Writer.java:249)
[INFO] 	at net.wasdev.wlp.test.dev.it.BaseDevTest.stopProcess(BaseDevTest.java:244)
[INFO] 	at net.wasdev.wlp.test.dev.it.BaseDevTest.cleanUpAfterClass(BaseDevTest.java:221)
[INFO] 	at net.wasdev.wlp.test.dev.it.BaseDevTest.cleanUpAfterClass(BaseDevTest.java:217)
[INFO] 	at net.wasdev.wlp.test.dev.it.BaseDevTest.cleanUpAfterClass(BaseDevTest.java:213)
[INFO] 	at net.wasdev.wlp.test.dev.it.DevTest.cleanUpAfterClass(DevTest.java:41)
[INFO] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[INFO] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[INFO] 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[INFO] 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
[INFO] 	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
[INFO] 	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
[INFO] 	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
[INFO] 	at org.junit.internal.runners.statements.RunAfters.invokeMethod(RunAfters.java:46)
[INFO] 	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:33)
[INFO] 	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
[INFO] 	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
[INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:367)
[INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:274)
[INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:238)
[INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:161)
[INFO] 	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
[INFO] 	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
[INFO] 	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
[INFO] 
```
</details>

<details>
 <summary>net.wasdev.wlp.test.dev.it.DevCopyProvidedDependenciesTest </summary>
```
```
[INFO] Tests run: 2, Failures: 1, Errors: 1, Skipped: 0, Time elapsed: 0.006 sec <<< FAILURE! - in net.wasdev.wlp.test.dev.it.DevCopyProvidedDependenciesTest
[INFO] net.wasdev.wlp.test.dev.it.DevCopyProvidedDependenciesTest  Time elapsed: 0 sec  <<< FAILURE!
[INFO] java.lang.AssertionError
[INFO] 	at org.junit.Assert.fail(Assert.java:87)
[INFO] 	at org.junit.Assert.assertTrue(Assert.java:42)
[INFO] 	at org.junit.Assert.assertTrue(Assert.java:53)
[INFO] 	at net.wasdev.wlp.test.dev.it.BaseDevTest.setUpBeforeClass(BaseDevTest.java:111)
[INFO] 	at net.wasdev.wlp.test.dev.it.DevCopyProvidedDependenciesTest.setUpBeforeClass(DevCopyProvidedDependenciesTest.java:35)
[INFO] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[INFO] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[INFO] 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[INFO] 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
[INFO] 	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
[INFO] 	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
[INFO] 	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
[INFO] 	at org.junit.internal.runners.statements.RunBefores.invokeMethod(RunBefores.java:33)
[INFO] 	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:24)
[INFO] 	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
[INFO] 	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
[INFO] 	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
[INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:367)
[INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:274)
[INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:238)
[INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:161)
[INFO] 	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
[INFO] 	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
[INFO] 	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
[INFO] 
[INFO] net.wasdev.wlp.test.dev.it.DevCopyProvidedDependenciesTest  Time elapsed: 0.006 sec  <<< ERROR!
[INFO] java.io.IOException: Stream closed
[INFO] 	at java.base/java.io.BufferedWriter.ensureOpen(BufferedWriter.java:107)
[INFO] 	at java.base/java.io.BufferedWriter.write(BufferedWriter.java:224)
[INFO] 	at java.base/java.io.Writer.write(Writer.java:249)
[INFO] 	at net.wasdev.wlp.test.dev.it.BaseDevTest.stopProcess(BaseDevTest.java:244)
[INFO] 	at net.wasdev.wlp.test.dev.it.BaseDevTest.cleanUpAfterClass(BaseDevTest.java:221)
[INFO] 	at net.wasdev.wlp.test.dev.it.BaseDevTest.cleanUpAfterClass(BaseDevTest.java:217)
[INFO] 	at net.wasdev.wlp.test.dev.it.BaseDevTest.cleanUpAfterClass(BaseDevTest.java:213)
[INFO] 	at net.wasdev.wlp.test.dev.it.DevCopyProvidedDependenciesTest.cleanUpAfterClass(DevCopyProvidedDependenciesTest.java:48)
[INFO] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[INFO] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[INFO] 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[INFO] 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
[INFO] 	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
[INFO] 	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
[INFO] 	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
[INFO] 	at org.junit.internal.runners.statements.RunAfters.invokeMethod(RunAfters.java:46)
[INFO] 	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:33)
[INFO] 	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
[INFO] 	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
[INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:367)
[INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:274)
[INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:238)
[INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:161)
[INFO] 	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
[INFO] 	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
[INFO] 	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
[INFO] 
```
</details>

<details>
 <summary>net.wasdev.wlp.test.dev.it.RunTest</summary>
```
[INFO] Running net.wasdev.wlp.test.dev.it.RunTest
[INFO] Tests run: 2, Failures: 1, Errors: 1, Skipped: 0, Time elapsed: 0.006 sec <<< FAILURE! - in net.wasdev.wlp.test.dev.it.RunTest
[INFO] net.wasdev.wlp.test.dev.it.RunTest  Time elapsed: 0.006 sec  <<< FAILURE!
[INFO] java.lang.AssertionError
[INFO] 	at org.junit.Assert.fail(Assert.java:87)
[INFO] 	at org.junit.Assert.assertTrue(Assert.java:42)
[INFO] 	at org.junit.Assert.assertTrue(Assert.java:53)
[INFO] 	at net.wasdev.wlp.test.dev.it.BaseDevTest.setUpBeforeClass(BaseDevTest.java:111)
[INFO] 	at net.wasdev.wlp.test.dev.it.BaseDevTest.setUpBeforeClass(BaseDevTest.java:80)
[INFO] 	at net.wasdev.wlp.test.dev.it.BaseDevTest.setUpBeforeClass(BaseDevTest.java:72)
[INFO] 	at net.wasdev.wlp.test.dev.it.RunTest.setUpBeforeClass(RunTest.java:43)
[INFO] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[INFO] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[INFO] 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[INFO] 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
[INFO] 	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
[INFO] 	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
[INFO] 	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
[INFO] 	at org.junit.internal.runners.statements.RunBefores.invokeMethod(RunBefores.java:33)
[INFO] 	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:24)
[INFO] 	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
[INFO] 	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
[INFO] 	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
[INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:367)
[INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:274)
[INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:238)
[INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:161)
[INFO] 	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
[INFO] 	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
[INFO] 	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
[INFO] 
[INFO] net.wasdev.wlp.test.dev.it.RunTest  Time elapsed: 0.006 sec  <<< ERROR!
[INFO] java.io.IOException: Stream closed
[INFO] 	at java.base/java.io.BufferedWriter.ensureOpen(BufferedWriter.java:107)
[INFO] 	at java.base/java.io.BufferedWriter.flushBuffer(BufferedWriter.java:117)
[INFO] 	at java.base/java.io.BufferedWriter.flush(BufferedWriter.java:256)
[INFO] 	at net.wasdev.wlp.test.dev.it.BaseDevTest.stopProcess(BaseDevTest.java:249)
[INFO] 	at net.wasdev.wlp.test.dev.it.BaseDevTest.cleanUpAfterClass(BaseDevTest.java:221)
[INFO] 	at net.wasdev.wlp.test.dev.it.BaseDevTest.cleanUpAfterClass(BaseDevTest.java:217)
[INFO] 	at net.wasdev.wlp.test.dev.it.RunTest.cleanUpAfterClass(RunTest.java:68)
[INFO] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[INFO] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[INFO] 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[INFO] 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
[INFO] 	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
[INFO] 	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
[INFO] 	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
[INFO] 	at org.junit.internal.runners.statements.RunAfters.invokeMethod(RunAfters.java:46)
[INFO] 	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:33)
[INFO] 	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
[INFO] 	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
[INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:367)
[INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:274)
[INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:238)
[INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:161)
[INFO] 	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
[INFO] 	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
[INFO] 	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
[INFO] 
```
</details>

<details>
 <summary>net.wasdev.wlp.test.dev.it.MultiModuleHotTestingTest</summary>
```
[INFO] Running net.wasdev.wlp.test.dev.it.MultiModuleHotTestingTest
[INFO] Tests run: 1, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 104.986 sec <<< FAILURE! - in net.wasdev.wlp.test.dev.it.MultiModuleHotTestingTest
[INFO] runTest(net.wasdev.wlp.test.dev.it.MultiModuleHotTestingTest)  Time elapsed: 82.583 sec  <<< FAILURE!
[INFO] java.lang.AssertionError
[INFO] 	at org.junit.Assert.fail(Assert.java:87)
[INFO] 	at org.junit.Assert.assertTrue(Assert.java:42)
[INFO] 	at org.junit.Assert.assertTrue(Assert.java:53)
[INFO] 	at net.wasdev.wlp.test.dev.it.MultiModuleHotTestingTest.runTest(MultiModuleHotTestingTest.java:63)
[INFO] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[INFO] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[INFO] 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[INFO] 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
[INFO] 	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
[INFO] 	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
[INFO] 	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
[INFO] 	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
[INFO] 	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
[INFO] 	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
[INFO] 	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
[INFO] 	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
[INFO] 	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
[INFO] 	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
[INFO] 	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
[INFO] 	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
[INFO] 	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
[INFO] 	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
[INFO] 	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
[INFO] 	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
[INFO] 	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
[INFO] 	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
[INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:367)
[INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:274)
[INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:238)
[INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:161)
[INFO] 	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
[INFO] 	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
[INFO] 	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
[INFO] 
```
</details>


<details>
<summary>net.wasdev.wlp.test.dev.it.MPStarterTest</summary>
```
2022-03-23T15:35:33.4860943Z [INFO] Running net.wasdev.wlp.test.dev.it.MPStarterTest
2022-03-23T15:37:37.1795086Z [INFO] Tests run: 2, Failures: 1, Errors: 1, Skipped: 0, Time elapsed: 123.686 sec <<< FAILURE! - in net.wasdev.wlp.test.dev.it.MPStarterTest
2022-03-23T15:37:37.1796112Z [INFO] net.wasdev.wlp.test.dev.it.MPStarterTest  Time elapsed: 123.685 sec  <<< FAILURE!
2022-03-23T15:37:37.1797037Z [INFO] java.lang.AssertionError: 
2022-03-23T15:37:37.1858705Z [INFO] Last 100 lines of log at /home/runner/work/ci.maven/ci.maven/liberty-maven-plugin/target/it/dev-it/target/../resources/mp-starter-project/logFile.txt:
2022-03-23T15:37:37.1859667Z [INFO] ===================== START =======================
2022-03-23T15:37:37.1860197Z [INFO] ====================== END ========================
2022-03-23T15:37:37.1860668Z [INFO] 
2022-03-23T15:37:37.1860937Z [INFO] 	at org.junit.Assert.fail(Assert.java:89)
2022-03-23T15:37:37.1861296Z [INFO] 	at org.junit.Assert.assertTrue(Assert.java:42)
2022-03-23T15:37:37.1861740Z [INFO] 	at net.wasdev.wlp.test.dev.it.BaseDevTest.startProcess(BaseDevTest.java:164)
2022-03-23T15:37:37.1862256Z [INFO] 	at net.wasdev.wlp.test.dev.it.BaseDevTest.startProcess(BaseDevTest.java:132)
2022-03-23T15:37:37.1862767Z [INFO] 	at net.wasdev.wlp.test.dev.it.BaseDevTest.startProcess(BaseDevTest.java:128)
2022-03-23T15:37:37.1863290Z [INFO] 	at net.wasdev.wlp.test.dev.it.BaseDevTest.setUpBeforeClass(BaseDevTest.java:123)
2022-03-23T15:37:37.1863823Z [INFO] 	at net.wasdev.wlp.test.dev.it.BaseDevTest.setUpBeforeClass(BaseDevTest.java:80)
2022-03-23T15:37:37.1864357Z [INFO] 	at net.wasdev.wlp.test.dev.it.BaseDevTest.setUpBeforeClass(BaseDevTest.java:76)
2022-03-23T15:37:37.1864902Z [INFO] 	at net.wasdev.wlp.test.dev.it.MPStarterTest.setUpBeforeClass(MPStarterTest.java:35)
2022-03-23T15:37:37.1865456Z [INFO] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
2022-03-23T15:37:37.1866125Z [INFO] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
2022-03-23T15:37:37.1866770Z [INFO] 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
2022-03-23T15:37:37.1867305Z [INFO] 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
2022-03-23T15:37:37.1867773Z [INFO] 	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
2022-03-23T15:37:37.1868324Z [INFO] 	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
2022-03-23T15:37:37.1868896Z [INFO] 	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
2022-03-23T15:37:37.1869485Z [INFO] 	at org.junit.internal.runners.statements.RunBefores.invokeMethod(RunBefores.java:33)
2022-03-23T15:37:37.1870052Z [INFO] 	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:24)
2022-03-23T15:37:37.1870599Z [INFO] 	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
2022-03-23T15:37:37.1871091Z [INFO] 	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
2022-03-23T15:37:37.1871515Z [INFO] 	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
2022-03-23T15:37:37.1872004Z [INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:367)
2022-03-23T15:37:37.1872625Z [INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:274)
2022-03-23T15:37:37.1873417Z [INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:238)
2022-03-23T15:37:37.1874007Z [INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:161)
2022-03-23T15:37:37.1874654Z [INFO] 	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
2022-03-23T15:37:37.1875336Z [INFO] 	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
2022-03-23T15:37:37.1875908Z [INFO] 	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
2022-03-23T15:37:37.1876277Z [INFO] 
2022-03-23T15:37:37.1876606Z [INFO] net.wasdev.wlp.test.dev.it.MPStarterTest  Time elapsed: 123.686 sec  <<< ERROR!
2022-03-23T15:37:37.1876982Z [INFO] java.io.IOException: Stream closed
2022-03-23T15:37:37.1877390Z [INFO] 	at java.base/java.lang.ProcessBuilder$NullOutputStream.write(ProcessBuilder.java:442)
2022-03-23T15:37:37.1877845Z [INFO] 	at java.base/java.io.OutputStream.write(OutputStream.java:157)
2022-03-23T15:37:37.1878310Z [INFO] 	at java.base/java.io.BufferedOutputStream.flushBuffer(BufferedOutputStream.java:81)
2022-03-23T15:37:37.1878807Z [INFO] 	at java.base/java.io.BufferedOutputStream.flush(BufferedOutputStream.java:142)
2022-03-23T15:37:37.1879272Z [INFO] 	at java.base/sun.nio.cs.StreamEncoder.implFlush(StreamEncoder.java:318)
2022-03-23T15:37:37.1879811Z [INFO] 	at java.base/sun.nio.cs.StreamEncoder.flush(StreamEncoder.java:153)
2022-03-23T15:37:37.1880257Z [INFO] 	at java.base/java.io.OutputStreamWriter.flush(OutputStreamWriter.java:251)
2022-03-23T15:37:37.1880699Z [INFO] 	at java.base/java.io.BufferedWriter.flush(BufferedWriter.java:257)
2022-03-23T15:37:37.1881163Z [INFO] 	at net.wasdev.wlp.test.dev.it.BaseDevTest.stopProcess(BaseDevTest.java:249)
2022-03-23T15:37:37.1881689Z [INFO] 	at net.wasdev.wlp.test.dev.it.BaseDevTest.cleanUpAfterClass(BaseDevTest.java:221)
2022-03-23T15:37:37.1882240Z [INFO] 	at net.wasdev.wlp.test.dev.it.BaseDevTest.cleanUpAfterClass(BaseDevTest.java:217)
2022-03-23T15:37:37.1882851Z [INFO] 	at net.wasdev.wlp.test.dev.it.BaseDevTest.cleanUpAfterClass(BaseDevTest.java:213)
2022-03-23T15:37:37.1883868Z [INFO] 	at net.wasdev.wlp.test.dev.it.MPStarterTest.cleanUpAfterClass(MPStarterTest.java:53)
2022-03-23T15:37:37.1884429Z [INFO] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
2022-03-23T15:37:37.1885015Z [INFO] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
2022-03-23T15:37:37.1885745Z [INFO] 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
2022-03-23T15:37:37.1886276Z [INFO] 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
2022-03-23T15:37:37.1886736Z [INFO] 	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
2022-03-23T15:37:37.1887283Z [INFO] 	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
2022-03-23T15:37:37.1887863Z [INFO] 	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
2022-03-23T15:37:37.1888444Z [INFO] 	at org.junit.internal.runners.statements.RunAfters.invokeMethod(RunAfters.java:46)
2022-03-23T15:37:37.1888999Z [INFO] 	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:33)
2022-03-23T15:37:37.1889493Z [INFO] 	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
2022-03-23T15:37:37.1889916Z [INFO] 	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
2022-03-23T15:37:37.1890407Z [INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:367)
2022-03-23T15:37:37.1894342Z [INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:274)
2022-03-23T15:37:37.1901803Z [INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:238)
2022-03-23T15:37:37.1908211Z [INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:161)
2022-03-23T15:37:37.1918218Z [INFO] 	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
2022-03-23T15:37:37.1925140Z [INFO] 	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
2022-03-23T15:37:37.1935178Z [INFO] 	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
2022-03-23T15:37:37.1936594Z [INFO] 
```
</details>

Looks like many of the test failures may be related:
```
[INFO] java.lang.AssertionError
[INFO] 	at org.junit.Assert.fail(Assert.java:87)
[INFO] 	at org.junit.Assert.assertTrue(Assert.java:42)
[INFO] 	at org.junit.Assert.assertTrue(Assert.java:53)
[INFO] 	at net.wasdev.wlp.test.dev.it.BaseDevTest.setUpBeforeClass(BaseDevTest.java:111)
```

--- a/liberty-maven-plugin/src/it/dev-it/resources/basic-dev-project/src/main/liberty/config/bootstrap.properties
+++ b/liberty-maven-plugin/src/it/dev-it/resources/basic-dev-project/src/main/liberty/config/bootstrap.properties
@@ -1,0 +1,1 @@
+com.ibm.ws.logging.trace.specification=config=all=enabled:*=info=enabled

--- a/liberty-maven-plugin/src/it/generate-features-it/src/test/java/net/wasdev/wlp/test/dev/it/GenerateFeaturesRestTest.java
+++ b/liberty-maven-plugin/src/it/generate-features-it/src/test/java/net/wasdev/wlp/test/dev/it/GenerateFeaturesRestTest.java
@@ -29,13 +29,6 @@ import org.junit.Test;
  * liberty:generate-features goal tests for various MicroProfile and Java EE versions
  * Test to ensure the binary scanner honours the version of MicroProfile and Java EE
  * specified in the pom.xml
- * When you use MicroProfile the binary scanner assumes you use the lastest subversion
- * in the version you specify:
- * You specify | binary scanner generates features in
- * MP 1.1-1.4  | MicroProfile 1.4
- * MP 2.0-2.2  | MicroProfile 2.2
- * MP 3.0-3.3  | MicroProfile 3.3
- * MP 4.0-4.1  | MicroProfile 4.1
  */
 public class GenerateFeaturesRestTest extends BaseGenerateFeaturesTest {
 
@@ -85,7 +78,7 @@ public class GenerateFeaturesRestTest extends BaseGenerateFeaturesTest {
     }
 
     @Test
-    public void mp2Test() throws Exception {
+    public void mp21Test() throws Exception {
         // Test Java EE 8.0 and MicroProfile 2.x
         // MicroProfile 2.1 uses cdi-2.0 so it requires EE8
         File pomFile = new File(tempProj, "pom.xml");
@@ -98,12 +91,12 @@ public class GenerateFeaturesRestTest extends BaseGenerateFeaturesTest {
 
         // verify that the correct features are in the generated-features.xml
         Set<String> features = readFeatures(newFeatureFile);
-        assertTrue(features.contains("mpRestClient-1.2"));
+        assertTrue(features.contains("mpRestClient-1.1"));
         assertTrue(features.contains("cdi-2.0"));
     }
 
     @Test
-    public void mp3Test() throws Exception {
+    public void mp30Test() throws Exception {
         // Test Java EE 8.0 and MicroProfile 3.x
         File pomFile = new File(tempProj, "pom.xml");
         replaceString("EE_VERSION", "8.0", pomFile);
@@ -115,12 +108,12 @@ public class GenerateFeaturesRestTest extends BaseGenerateFeaturesTest {
 
         // verify that the correct features are in the generated-features.xml
         Set<String> features = readFeatures(newFeatureFile);
-        assertTrue(features.contains("mpRestClient-1.4"));
+        assertTrue(features.contains("mpRestClient-1.3"));
         assertTrue(features.contains("cdi-2.0"));
     }
 
     @Test
-    public void mp4Test() throws Exception {
+    public void mp41Test() throws Exception {
         // Test Java EE 8.0 and MicroProfile 4.x
         File pomFile = new File(tempProj, "pom.xml");
         replaceString("EE_VERSION", "8.0", pomFile);

--- a/liberty-maven-plugin/src/it/generate-features-it/src/test/java/net/wasdev/wlp/test/dev/it/GenerateFeaturesTest.java
+++ b/liberty-maven-plugin/src/it/generate-features-it/src/test/java/net/wasdev/wlp/test/dev/it/GenerateFeaturesTest.java
@@ -232,26 +232,4 @@ public class GenerateFeaturesTest extends BaseGenerateFeaturesTest {
         }
         return str.toString();
     }
-
-    @Test
-    public void mpVersionTest() throws Exception {
-       assertEquals(GenerateFeaturesMojo.getMPVersion("mpHealth-2.0"), 3);
-       assertEquals(GenerateFeaturesMojo.getMPVersion("mpconfig-1.0"), 1);
-       assertEquals(GenerateFeaturesMojo.getMPVersion("mpconfig-1.3"), 2);
-       assertEquals(GenerateFeaturesMojo.getMPVersion("mpconfig-1.4"), 3);
-       assertEquals(GenerateFeaturesMojo.getMPVersion("mprestclient-1.0"), 1);
-       assertEquals(GenerateFeaturesMojo.getMPVersion("mprestclient-1.1"), 1);
-       assertEquals(GenerateFeaturesMojo.getMPVersion("mprestclient-1.2"), 2);
-       assertEquals(GenerateFeaturesMojo.getMPVersion("mprestclient-1.3"), 3);
-       assertEquals(GenerateFeaturesMojo.getMPVersion("mprestclient-1.4"), 3);
-       assertEquals(GenerateFeaturesMojo.getMPVersion("mpjwt-1.0"), 1);
-       assertEquals(GenerateFeaturesMojo.getMPVersion("mpjwt-1.1"), 3);
-       assertEquals(GenerateFeaturesMojo.getMPVersion("mpjwt-1.2"), 4);
-       // Error testing
-       assertEquals(GenerateFeaturesMojo.getMPVersion("mprestclientX-1.5"), 0);
-       assertEquals(GenerateFeaturesMojo.getMPVersion("mprestclient1.5"), 0);
-       assertEquals(GenerateFeaturesMojo.getMPVersion("mprestclient-1.5a"), 0);
-       assertEquals(GenerateFeaturesMojo.getMPVersion("mprestclient-10"), 0);
-       assertEquals(GenerateFeaturesMojo.getMPVersion("Xmprestclient-1.0"), 0);
-    }
 }

